doc/7/controllers/index/create/index.md

```diff
@@ -42,4 +42,11 @@ Resolves if the index does not already exists
 
 ## Usage
 
+:::: tabs
+::: tab yourTabName
```

Comment:
does this really render to `yourTabName` and `anotherTab`, or is this just a copy-paste error?

Reply:
It was an experiment to include TS snippet but it will be done later since it's a lot of work
(removed)